Makefile

```diff
@@ -112,12 +112,12 @@ install: compile
 	$(MAKE) install_man
 
 install_man:
-	$(Q) mkdir -p $(PREFIX)/share/man/man1
+	$(Q) mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
```

Comment:
Shouldn't this be `$(DESTDIR)/$(PREFIX)` too? And what would happen if both are empty? Would it install it at root?